docker/.env.example

```
@@ -2,4 +2,29 @@ COMPOSE_PROJECT_NAME=your_project_name_here
 
 # PostgreSQL 설정
 POSTGRES_USER=your_user_here
-POSTGRES_DB=your_db_here
\ No newline at end of file
+POSTGRES_DB=your_db_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_DB key should go before the POSTGRES_USER key

(UnorderedKey)